fix: update section headings to align with v1 documentation (#1343)
auth0/docs-v2: main/docs/api/authentication/multi-factor-authentication/add-an-authenticator.mdx

  The phone number to use for SMS or Voice. Required if `oob_channels` includes `sms` or `voice`.
</ParamField>

## Response Schema

<ResponseSchema>
<ResponseField name="oob_code" type="string">

>   The code used for out-of-band authentication.

Reviewer: ```suggestion
  The code for out-of-band authentication.
```

</ResponseField>

<ResponseField name="binding_method" type="string">

>   Indicates the binding method used.

Reviewer: ```suggestion
  Indicates the binding method.
```

</ResponseField>

<ResponseField name="authenticator_type" type="string">

>   The type of authenticator added.

Reviewer: ```suggestion
  The authenticator type.
```

</ResponseField>

<ResponseField name="oob_channels" type="string">

>   The OOB channels used.

Reviewer: ```suggestion
  The OOB channels the authenticator uses.
```

</ResponseField>

<ResponseField name="recovery_codes" type="array">

>   An array of recovery codes generated for the user. Included only the first time an authenticator is added.

Reviewer: ```suggestion
  An array of recovery codes Auth0 generates for the user. When a user adds an authenticator, recovery codes display only once.
```
